src/aequitas_webapp/views.py

```diff
@@ -198,6 +198,6 @@ def report(dirname, name, reportid):
     )
 
 
-@app.route('/example.html', methods=['GET'])
+@app.route('/example', methods=['GET'])
```

Comment:
any reason to strip this off of the URL path? the resource is just the read-only HTML page, after all. the site can use these or not, but a bunch of its other, similar resources identify themselves this way.

Reply:
@anisfeld

Reply:
I don't know why I did that. I'll adjust it.

Reply:
Done.

Reply:
not a big deal obviously, but cool. thanks :smile: